src/components/Greeting.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+function Greeting() {
+  const greeting = useSelector((state) => state);
+  return (
+    <h2>{greeting.message}</h2>
```

Comment:
- [ ] I have found out that you are calling the wrong property on the greeting api and I think changing `greeting.message` to `greeting.text` will fix the issue mentioned above :+1: 

![image](https://github.com/alicemirigo92/react-frontend/assets/92303891/4d1d939b-47e2-4869-8ecd-15cbf9442109)

Reply:
@Owusu-Desmond Thanks for review. I implemented the required changes